Lib/test/test_ssl.py

```diff
@@ -1329,7 +1329,7 @@ def test_load_verify_cadata(self):
         with self.assertRaisesRegex(ssl.SSLError, "not enough data"):
             ctx.load_verify_locations(cadata=b"broken")
 
-
+    @unittest.skipIf(platform.win32_is_iot() and 'python_d' in sys.executable, "Crashes on debug python builds on Windows IoT Core")
```

Comment:
I thought we had a flag for debug builds somewhere in support? Rather than checking the name.

Reply:
I see a lot of checks like this: Py_DEBUG = hasattr(sys, 'gettotalrefcount')
but I haven't found a shared property/method yet.